docs/demo/gaiden-css/components/card/profile.html

```diff
@@ -16,257 +16,240 @@
-                  <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
-                </li>
-                <li class="rating__star-item">
-                  <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
-                </li>
-                <li class="rating__star-item">
-                  <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
-                </li>
-                <li class="rating__star-item">
-                  <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
-                </li>
-              </ul>
-              (<span class="rating__number card-citation__rating-number">19 recomendações</span>)
-            </div>
-          </header>
-
-          <div class="card-profile__content">
-            <p class="card-profile__text">
-              Trabalhamos com reformas em geral. Temos fábrica de esquadrias de alumínio. Box e sacadas, fechamentos de coberturas. Policarbo...
-            </p>
+        <header class="card-profile__header">
+          <h3 class="card-profile__title">Carlos - Eletricista</h3>
+
+          <div class="card-profile__rating">
+            <ul class="rating__stars">
+              <li class="rating__star-item">
+                <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
+              </li>
+              <li class="rating__star-item">
+                <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
+              </li>
+              <li class="rating__star-item">
+                <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
+              </li>
+              <li class="rating__star-item">
+                <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
+              </li>
+              <li class="rating__star-item">
+                <img alt="Star icon" width="14" height="14" src="/demo/gaiden-css/images/star-full.svg">
+              </li>
+            </ul>
+            (<span class="rating__number card-citation__rating-number">19 recomendações</span>)
           </div>
+        </header>
+
+        <div class="card-profile__content">
+          <p class="card-profile__text">
+            Trabalhamos com reformas em geral. Temos fábrica de esquadrias de alumínio. Box e sacadas, fechamentos de coberturas. Policarbo...
+          </p>
         </div>
 
-        <div>
-          <div class="card-profile__highlight space-element-medium">
+        <footer class="card-profile__footer">
+          <div class="card-profile__highlight">
             <h4 class="card-profile__light-title">Avaliação mais recente:</h4>
 
-            <p class="card-profile__light-text">
+            <p class="card-profile__text card-profile__text--italic">
```

Comment:
I created a class that modifies an existing class that had the same styles.

Reply:
Very well! 👍

Reply:
We prefer to not use a modifier that specifies properties of CSS. My suggestion is a `card-profile__text-citation`.

Reply:
@eduardojmatos I understand. But would we stay with those class in markup?

`class="cad-profile__text card-profile__text-citation"`

Or will we choose to repeat the same text properties that we already use in another element?

Reply:
Sorry.. Correcting: `card-profile__text--citation`

Reply:
Nice!